src/domain/attendanceList/attendeeList/AttendeeList.tsx

```diff
@@ -106,6 +107,7 @@ const AttendeeList: React.FC<Props> = ({ registration }) => {
 
   return (
     <>
+      <SearchStatus count={attendees.length} loading={false} />
```

Comment:
Should this be updating the accessibility text after a user filters the attendee list using the search input? At the moment it seems to set only the initial search result and leave it at that.

Reply:
This is not intentional. I changed the behaviour so that accessibility output message is changed when filtered attendees count changes